examples/LoRaWAN/LoRaWAN_ABP/configABP.h

```diff
@@ -46,6 +48,11 @@ const uint8_t subBand = 0;  // For US915, change this to 2, otherwise leave on 0
     #pragma message ("Link required on board")
     SX1276 radio = new Module(8, 3, 4, 6);
 
+// Pi_Pico_1262_LoRa
+#elif defined(Pi_Pico_1262_LoRa)
+  #pragma message ("Pi_Pico_1262_LoRa")
+     // fix from https://github.com/jgromes/RadioLib/issues/729
+  SX1262 radio = new Module(3, 20, 15, 2, SPI1, RADIOLIB_DEFAULT_SPI_SETTINGS);
```

Comment:
I don't think it can be called a "fix" when you are using the library as intended - what exactly is being fixed here?

Having to specify which SPI bus is used to connect to the module is a very basic information that even has its own section on the Wiki and in the troubleshooting guide. It's not something I want to hide from the user, if they leave everything at the default value and then *insert surprised Pikachu face* when it doesn't work then that suggests they know very little about the hardware they are using.